Krytskaya.feature

```diff
@@ -0,0 +1,53 @@
+Feature: 1.1. Contacts
+
+Background:
+    Given user clicked on Notifications "Contacts" 
+
+Scenario Outline: 1.1.1 All external links should be active and right
+    Given user scrolled Contacts to bottom app bars
+    When user clicked on "<Button>" icon
+    Then open a new window in browser 
+    And open an external link <Go to>:
+        | Button   | Go to                                     |
+        | site     | https://www.quantori.com/                 |
+        | mail to  | career@quantori.com                       |
+        | hh       | https://hh.ru/employer/3343960            |
+        | facebook | https://www.facebook.com/quantoricom/     |
+        | instagram| https://instagram.com/quantori_community  |
+        | vk       | https://vk.com/quantori                   |
+        | linkedin | https://www.linkedin.com/company/quantori |
+
+Scenario Outline: 1.1.2 All internal notifications links should be active and right
+    Given user scrolled Contacts to the top app bars
+    When user clicked on "<Notification>" icon
+    Then open an internal link <Go to>:
+        | Notification      | Go to                                     |
+        | About             | https://career.quantori.com/ru/about      |
+        | Open job positions| https://career.quantori.com/ru/positions  |
+        | Contact           | https://career.quantori.com/ru/contact    |
+
+Scenario Outline: 1.1.3 An internal link to the title page should be active and right

```

Comment:
In this case we should use Scenario (not Scenario Outline) because we have only one value in the examples.

Reply:
Ok